gqcp/include/Basis/Transformations/SimpleTransformation.hpp

```diff
@@ -78,6 +84,15 @@ class SimpleTransformation:
         T {T} {}
 
 
+    /**
+     *  Construct a `SimpleTransformation` from the transformation matrix (associated with a set of orbital rotation generators) that it encapsulates.
+     * 
+     *  @param orbital_rotation_generators                The orbital rotation generators from which a transformation so-called `kappa matrix` is constructed.
+     */
+    SimpleTransformation(const OrbitalRotationGeneratorType& orbital_rotation_generators) :
```

Comment:
```suggestion
    SimpleTransformation(const OrbitalRotationGeneratorType& kappa) :
```

As a general guideline, it's not always the best idea to give a variable the name of its type.

Reply:
Allright, I'll keep that in mind from now on!